docs/style.css

```diff
@@ -339,6 +339,13 @@ tbody td.wikidata div.viewlink {
   color: #2a2a2a;
   background: #88c;
 }
+
+/* SSL lock / unlock icons */
+.viewlink svg.fa-lock   { margin-left: 0.3em; color: green; background: white; }
+.viewlink svg.fa-unlock { margin-left: 0.3em; color: red;   background: white; }
+.dark .viewlink svg.fa-lock   { margin-left: 0.3em; color: green; background: #2a2a2a; }
+.dark .viewlink svg.fa-unlock { margin-left: 0.3em; color: red;   background: #2a2a2a; }
```

Comment:
I'm very late to the party @UKChris-osm , but I think these look a lot better without the white/dark background, so they're naturally transparent and match the underlying page colour; maybe that has changed since this went in?